tensorboard/components/tf_paginated_view/tf-category-paginated-view.html

```diff
@@ -154,6 +164,9 @@
         line-height: 1;
         box-shadow: 0 1px 5px rgba(0, 0, 0, 0.2);
         padding: 10px 15px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
       }
```

Comment:
there are so many counter examples but prefer to sort the CSS a-z (as per style guide but it is hard to enforce that)

Reply:
Done, sorted this rule.

I would really prefer updating our style guide, a-z makes it much harder to read, in my opinion.  "align" and "justify" are very related to "display", and seeing them sorted a-z makes it harder to reason about layout.

Reply:
For posterity, this the styleguide I am referring to: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Thanks for the link.  I realize that we should follow a guide, but since the codebase is already unsorted, and we have no enforcement atm, I would prefer to defer style-guide-alignment to later.